images/macos/provision/core/python.sh

```diff
@@ -6,11 +6,7 @@ echo "Installing Python Tooling"
 
 echo "Brew Installing Python 3"
 # Workaround to have both 3.8 & 3.9(which required by some brew formulas) in the system, but only 3.8 is linked
-brew install python@3.8
 brew install python@3.9
```

Comment:
Maybe just `brew install python3` to stay on the latest version in any cases?
@maxim-lobanov @AlenaSviridenko thoughts?

Reply:
I personally prefer specifying the exact version as it looks more clear for everyone what is the default version.